pkg/controller/registry/resolver/resolver_test.go

```diff
@@ -1041,4 +1186,4 @@ func genOperator(name, version, replaces, pkg, channel, catalogName, catalogName
 func stripBundle(o *Operator) *Operator {
 	o.bundle = nil
 	return o
-}
\ No newline at end of file
+}
```

Comment:
lot of noise towards the end of this file too.

Reply:
some format should be fine right. nothing wrong with formating it when it was not properly formated the first time.

Reply:
The problem with this different people have different formatting settings in their code editor. So every time this code base is touched, the formatting will just go back and forth between these options. 
This PR isn't about correcting the formatting is my point. 
If we want to adopt a convention for the project so that everyone contributing to this project have the same formatting settings set for their code editor, that can be done in a separate PR.

Reply:
Also, whoever formatted the code the way it was before must have been of the opinion that that was the right formatting for the code, while we're saying no I disagree and it has to be changed in this PR.

Reply:
it's not a settign thing if there no new line at the end. other format input, yes.

Reply:
Right my comment is on the other formatting bits.